### DISCUSSION TITLE
Store recordings on a network path

### DISCUSSION THREAD

**Discussion by dfigus** [dfigus/addon-tvheadend#112] [General]
Hi @rucati,

to discuss the network / smb topic from #103:

>ALSO it includes a config-path so you can add a smb-path to store recordings, apart from your local disk.
>  
>@dfigus can a config-option to add an export-dir in the docker environment be added? 
>(Like the above link) This allows recordings to be saved on a NAS.
_Originally posted by @rucati in https://github.com/dfigus/addon-tvheadend/issues/103#issuecomment-1947878933_

I think you can already do this with the [network storage feature of HA](https://www.home-assistant.io/common-tasks/os/#network-storage). The add-on maps `/media` and `/share` and in the network storage options you can choose whether the network location shall be mounted as a subfolder on `media` or `share`. Afterwards you would need to update your TVH recordings location to use the new subfolder in `/media` or `/share`.

Can you try this?

**Comment by obert0** [comment]
Hi, I explore the addons of @alexbegium in his repo: https://github.com/alexbelgium/hassio-addons and he have a nice set of script. I use his Jellyfin addon and the smb mapping work well. He also have a function to map local drive that can be a nice feature!
@dfigus I try the network storage feature of HA and it work very well! I think this can be a better to handle network mapping. I will try to record to this location!
